### ISSUE TITLE
You can use pAI voodoo to respawn on arrivals

### ISSUE THREAD

**Issue by Cyberboss** [tgstation/tgstation#27804]
```
12:46:33: Reply PM from-Your(Mother): I died as Janitor - turned into pAI - got uninstalled as pAI - and then went back into the body, rejoined the game and was able to select a new role to play, Im an assistant now
```

@kevinz000  pls

**Comment by Marflowerose** [comment]
Possibly same pAI, I deconstructed it using destructive analyzer if that matters.

**Comment by ChangelingRain** [comment]
Bonus part of the report:
`AND WHEN I REJOINED IT MADE EVERYONE UNABLE TO USE THE DEFAULD SAY COMMAND`

**Comment by Shadowlight213** [comment]
Might have been fixed with #27812
